tt_metal/llrt/llrt.hpp

```diff
@@ -96,15 +96,11 @@ void launch_erisc_app_fw_on_core(chip_id_t chip, CoreCoord core);
 void print_worker_cores(chip_id_t chip_id = 0);
 
 inline bool is_worker_core(const CoreCoord &core, chip_id_t chip_id) {
-    const metal_SocDescriptor &soc_desc = tt::Cluster::instance().get_soc_desc(chip_id);
-    return std::find(soc_desc.physical_workers.begin(), soc_desc.physical_workers.end(), core) !=
-           soc_desc.physical_workers.end();
+    return tt::Cluster::instance().is_worker_core(core, chip_id);
```

Comment:
can we just get rid of `is_worker_core` and `is_ethernet_core` below and use the cluster apis

Reply:
Done